### ISSUE TITLE
Clusterous timing out

### ISSUE THREAD

**Issue by jscamac** [sirca/clusterous#152]
Hi guys,
I've ran into another issue where clusterous times out when trying to run `clusterous run clusterous_env.yml` with > 10 instances. 

The error I get is:

```
Checking for Docker images...
Copying files...
Starting 136 instances of engine
Starting 1 instance of redis
Timed out waiting for components to launch
One or more containers are either have problems or are are taking very long to start
Could not launch all components
Launched 1 components: redis
```

My work around solution (thanks to @lolo-fernandez-sirca) is to allow this error to occur and then run
`clusterous quit`
and then rerun
`clusterous run clusterous_env.yml`

clusterous then correctly launches the environment as normal.

Below I will try and supply you with a minimum working example:
`mycluster.yml` looks like:

```
cluster_name: mortality_cluster
parameters:
    master_instance_type: t2.micro
    worker_instance_type: c4.2xlarge
    worker_count: 17
```

`clusterous_env.yml` looks like:

```
name: mortality    # environment name
environment:  # This is the main section where you describe the environment
  # Copy any necessary folders to the cluster shared storage.
  # The folder name is relative to the location of this environment file
  copy:
    - "precompile"
    - "R"
    - "docker"
  # Section that describes your application components
  components:
    redis:     # A component named "master"
      machine: master       # Run on the machine named "master"
      cpu: auto             # Evenly share CPU with any other components running on this machine
      image: redis   # Name of the redis image to run
      cmd: "/usr/local/bin/redis-server"   # The command to be run in the container
      ports: "6379"        # Expose redis port 6379 as 6379 on the host
    engine:   # Component named "engine". These are the "worker" processes
      machine: worker       # Run on machine(s) named "worker"
      depends: redis       # Run only after "master" component has been run
      cpu: 1              # 1 CPU per running instance of this component
      count: auto           # Automatically run as much as possible on all machines of this type
      image: traitecoevo/mortality_bci 
      cmd: "/bin/bash /home/data/docker/worker.sh"   # Launch command
  # Once the above environment is running, create an SSH tunnel to one of the
  # components. This is typically used to expose a web UI or a queue on the local
  # machine. This section is optional.
  expose_tunnel:
    # The component to connect to and the ports. In the format:
    # local_port:component_name:source_port
    service: 6379:redis:6379
    # An optional message to display to the user once the port is exposed
    # The "{url}" field is replaced by an HTTP url to the tunnel
    message: "To access redis, use this URL: {url}"
```

The above environment file pulls to docker images `redis` for the master and `traitecoevo/mortality_bci` for the workers. Both are available on dockerhub. It also copies over three local folders `R` which contains some source code to run functions and models; `precompile` which simply contains C compiled Bayesian models and `docker` which contains the clusterous_env.yml, `my cluster.yml` and a shell script `worker.sh`

which is this:

```
if [ -z "$REDIS_HOST" ]; then
    REDIS_HOST=redis.marathon.mesos
    REDIS_PORT=6379
else
    REDIS_HOST=$(env | grep '^REDIS.*TCP_ADDR=' | sed  's/^.*=//')
    REDIS_PORT=$(env | grep '^REDIS.*TCP_PORT=' | sed  's/^.*=//')
fi

rrqueue_worker --redis-host ${REDIS_HOST} --redis-port ${REDIS_PORT} rrq
```

If it is easier I'm happy to share a zip of the folder ( I think @lolo-fernandez-sirca already has it)
